### PR TITLE
Builtin typecast

### PR DESCRIPTION
**Pull request by aannleax**
Automatically cast numeric builtins to to double if needed

### CODE REVIEW

File: nemo/src/execution/planning/plan_head_restricted.rs [hunk 1]
```diff
@@ -113,6 +113,7 @@ impl HeadStrategy for RestrictedChaseStrategy {
         current_plan: &mut SubtableExecutionPlan,
         variable_translation: &VariableTranslation,
         node_matches: ExecutionNodeRef,
+        _node_aggregates: Option<ExecutionNodeRef>,
```
Comment (monsterkrampe) [inline nemo/src/execution/planning/plan_head_restricted.rs:116]: Why is this here? The underscore seems to indicate that this is not used(?)

**Review by monsterkrampe — APPROVED** [review]
Just one question about a line that might be obsolete.

File: nemo/src/execution/planning/plan_head_restricted.rs (hunk 1 above)
Comment (aannleax) [reply nemo/src/execution/planning/plan_head_restricted.rs:116]: Aggregation unterstützen wir bislang nicht zusammen mit existentiellen Regeln
